src/IB/FEMechanicsBase.cpp

```diff
@@ -579,7 +579,7 @@ FEMechanicsBase::doInitializeFEData(const bool use_present_data)
         // Set up boundary conditions.  Specifically, add appropriate boundary
         // IDs to the BoundaryInfo object associated with the mesh, and add DOF
         // constraints for the nodal forces and velocities.
-        const MeshBase& mesh = equation_systems.get_mesh();
+        MeshBase& mesh = equation_systems.get_mesh();
```

Comment:
Do we need to lose the `const` here?

Reply:
Yes, because we call `boundary_info.add_node(node, dirichlet_bdry_ids)` below.